### ISSUE TITLE
Change rc filepath

### ISSUE THREAD

**Issue by meymard** [fcambus/ansiweather#58]
Hi,
I would like have many rc config file. I have already fixed this issue.
I have also added a config for change default color.

My patch :
``` patch
diff --git a/ansiweather b/ansiweather
index 6bdc60b..b4ad6a7 100755
--- a/ansiweather
+++ b/ansiweather
@@ -19,7 +19,12 @@
 
 LC_NUMERIC=C
 
-config_file=~/.ansiweatherrc
+if [ $# -gt 0 ]
+then
+    config_file=$1
+else
+    config_file=~/.ansiweatherrc
+fi
 
 function get_config {
 	ret=""
@@ -110,6 +115,7 @@ done
 
 #### [ Colors and characters ]#################################################
 
+initialize_color=$(get_config "initialize_color" || echo "\033[0m")
 background=$(get_config "background" || echo "\033[44m")
 text=$(get_config "text" || echo "\033[36;1m")
 data=$(get_config "data" || echo "\033[33;1m")
@@ -348,7 +354,7 @@ then
 			output="$output$dashes "
 		fi
 	done
-	output="$output\033[0m"
+	output="$output$initialize_color"
 	echo -e "$output"
 else
 	if [ $symbols = true ]
@@ -362,7 +368,7 @@ else
 		output="$output $dashes$text $sunrise_text $delimiter$data $sunrise_time $dashes$text $sunset_text $delimiter$data $sunset_time"
 	fi
 
-	output="$output \033[0m"
+	output="$output$initialize_color"
 
 	echo -e "$output"
 fi
```
--
Sorry for my english,
Marc

**Comment by fcambus** [comment]
Merged #61 which allow to specify config file path.

**Comment by fcambus** [comment]
Closing as issue was solved by #61.
